ocd_frontend/wsgi.py

```diff
@@ -24,3 +27,6 @@ def serve_media(filename):
         base_dir = os.path.join(application.app.config.get('THUMBNAILS_DIR'),
                                 os.path.dirname(filename))
         return send_from_directory(base_dir, os.path.basename(filename))
+
+if __name__ == '__main__':
+    run_simple('0.0.0.0', 5000, application, processes=8, use_reloader=True, use_debugger=True)
```

Comment:
The wsgi server is started with 8 processes, but `ps aux` only shows the wsgi.py file. Maybe it can be done in another way.